### DISCUSSION TITLE
Horizontal scrollbar

### DISCUSSION THREAD

**Discussion by sund3RRR** [zed-industries/zed#10005] [Feature Requests]
How to enable horizontal scrollbar in edit window?

**Comment by 0x2CA** [comment]
As far as I know there is no such feature now, using extra long lines of code with no line breaks is a disaster, usually constrained to 80 columns or editor width soft wrap

**Reply by sund3RRR** [reply]
I know but there are some points where it might be useful.
For example, when I copy some big text to write a comment, I need to wrap lines and cut the text, but without a horiz scrollbar this is less convenient.

**Reply by sund3RRR** [reply]
Other code editors have this feature. I don't understand why devs don't add this feature at least as an option rather than default unless someone needs a horiz scrollbar to save 5px of vertical space.

**Reply by sund3RRR** [reply]
I just hope that dev's opinion is not like this https://github.com/alacritty/alacritty/issues/6756#issuecomment-1455795899
By the way it is the main reason why I don't use alacritty

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
